src/app/ui/components/containers/footers/available-balance.tsx

```diff
@@ -18,7 +18,11 @@ export function AvailableBalance({
         <styled.span color="ink.text-subdued" textStyle="caption.01">
           Available balance
         </styled.span>
-        <BasicTooltip label={balanceTooltipLabel} side="top">
+        <BasicTooltip
+          label={balanceTooltipLabel}
+          side="top"
+          onClickTrigger={e => e.preventDefault()}
```

Comment:
If you set `asChild`, it won't be a button and would avoid this

IMO `asChild` should be default (regarldess of accessibility concerns)

Reply:
Thanks, I didn't know that. It's much cleaner. 

I held off making it default for now